<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimerFrameItem1</class>
 <widget class="QWidget" name="TimerFrameItem1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1442</width>
    <height>69</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QTimeEdit{
background-color: rgb(219, 209, 231);
color:blue;
	font: 75 14pt &quot;宋体&quot;;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="labelGroupName">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">
font: 75 12pt &quot;微软雅黑&quot;;
 background-color: gray;
    border-style: outset;
    border-width: 2px;

    border-color: black;
    font: bold 14px;
    padding: 6px;</string>
     </property>
     <property name="text">
      <string>gfgfl</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>36</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>36</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>一段：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>36</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>36</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>二段：</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTimeEdit" name="timeEdit1_1"/>
       </item>
       <item>
        <widget class="QTimeEdit" name="timeEdit2_1"/>
       </item>
       <item>
        <widget class="QTimeEdit" name="timeEdit3_1"/>
       </item>
       <item>
        <widget class="QTimeEdit" name="timeEdit4_1"/>
       </item>
       <item>
        <widget class="QTimeEdit" name="timeEdit5_1"/>
       </item>
       <item>
        <widget class="QTimeEdit" name="timeEdit6_1"/>
       </item>
       <item>
        <widget class="QTimeEdit" name="timeEdit7_1"/>
       </item>
       <item>
        <widget class="QTimeEdit" name="timeEdit8_1"/>
       </item>
       <item>
        <widget class="QTimeEdit" name="timeEdit9_1"/>
       </item>
       <item>
        <widget class="QTimeEdit" name="timeEdit10_1"/>
       </item>
       <item>
        <widget class="QTimeEdit" name="timeEdit11_1"/>
       </item>
       <item>
        <widget class="QTimeEdit" name="timeEdit12_1"/>
       </item>
       <item>
        <widget class="QTimeEdit" name="timeEdit13_1"/>
       </item>
       <item>
        <widget class="QTimeEdit" name="timeEdit14_1"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTimeEdit" name="timeEdit1_2">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTimeEdit" name="timeEdit2_2"/>
       </item>
       <item>
        <widget class="QTimeEdit" name="timeEdit3_2"/>
       </item>
       <item>
        <widget class="QTimeEdit" name="timeEdit4_2"/>
       </item>
       <item>
        <widget class="QTimeEdit" name="timeEdit5_2"/>
       </item>
       <item>
        <widget class="QTimeEdit" name="timeEdit6_2"/>
       </item>
       <item>
        <widget class="QTimeEdit" name="timeEdit7_2"/>
       </item>
       <item>
        <widget class="QTimeEdit" name="timeEdit8_2"/>
       </item>
       <item>
        <widget class="QTimeEdit" name="timeEdit9_2"/>
       </item>
       <item>
        <widget class="QTimeEdit" name="timeEdit10_2"/>
       </item>
       <item>
        <widget class="QTimeEdit" name="timeEdit11_2"/>
       </item>
       <item>
        <widget class="QTimeEdit" name="timeEdit12_2"/>
       </item>
       <item>
        <widget class="QTimeEdit" name="timeEdit13_2"/>
       </item>
       <item>
        <widget class="QTimeEdit" name="timeEdit14_2"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0);</string>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
